subprojects/tooling-api/tooling-api.gradle

```diff
@@ -62,6 +63,31 @@ task shadedJar(type: ShadedJar) {
     shadowPackage = "org.gradle.internal.impldep"
 }
 
+task toolingApiShadedJar(type: Zip) {
+    dependsOn shadedJarWithoutVersion, project(':versionInfo').createBuildReceipt
```

Comment:
Copy and paste error. Thanks for spotting is. I removed the duplication and fixed the mistake.